mmv1/third_party/terraform/services/monitoring/resource_monitoring_dashboard_test.go

```diff
@@ -117,6 +117,15 @@ func TestAccMonitoringDashboard_update(t *testing.T) {
 				ImportStateVerify:       true,
 				ImportStateVerifyIgnore: []string{"project"},
 			},
+			{
+				Config: testAccMonitoringDashboard_gridLayoutUpdate(),
+			},
+			{
+				ResourceName:            "google_monitoring_dashboard.dashboard",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"project"},
```

Comment:
What do each of these fields e.g. `ImportState` do? I read what appears to be the relevant document at https://googlecloudplatform.github.io/magic-modules/develop/test/ - perhaps I am missing something?

Reply:
These fields are used to test if importing the resource works or not.

`ImportState` triggers the import testing mode. `ImportStateVerify` will compare the state after import with the server side response. `ImportStateVerifyIgnore` will not compare the ignored fields.